static/css/s2.css

```diff
@@ -12983,6 +12983,151 @@ span.ref-link-color-3 {color: blue}
     width: 67%;
   }
 }
+
+.product {
+  max-width: 600px;
+}
+
+#productsPageContent {
```

Comment:
Isnt there something more globally defined?

Reply:
This was the existing pattern used for the super-about page, which I followed. Curious if there's a better approach.